website/content/docs/secrets/consul.mdx

```diff
@@ -27,128 +27,137 @@ management tool.
     By default, the secrets engine will mount at the name of the engine. To
     enable the secrets engine at a different path, use the `-path` argument.
 
-1. Vault can bootstrap the ACL system of your Consul cluster if it has
-   not already been done. In this case, you only need the address of your
-   Consul cluster to configure the Consul secret engine:
-
-    ```text
-    $ vault write consul/config/access \
-        address=127.0.0.1:8500
-    Success! Data written to: consul/config/access
-    ```
-
-    If you have already bootstrapped the ACL system of your Consul cluster, you
-    will need to give Vault a management token:
-
-    In Consul versions below 1.4, acquire a [management token][consul-mgmt-token] from Consul, using the
-    `acl_master_token` from your Consul configuration file or another management
-    token:
-
-    ```sh
-    $ curl \
-        --header "X-Consul-Token: my-management-token" \
-        --request PUT \
-        --data '{"Name": "sample", "Type": "management"}' \
-        https://consul.rocks/v1/acl/create
-    ```
-
-    Vault must have a management type token so that it can create and revoke ACL
-    tokens. The response will return a new token:
-
-    ```json
-    {
-    "ID": "7652ba4c-0f6e-8e75-5724-5e083d72cfe4"
-    }
-    ```
-
-    For Consul 1.4 and above, use the command line to generate a token with the appropriate policy:
-
-    ```shell-session
-    $ CONSUL_HTTP_TOKEN="<management-token>" consul acl token create -policy-name="global-management"
-    AccessorID:   865dc5e9-e585-3180-7b49-4ddc0fc45135
-    SecretID:     ef35f0f1-885b-0cab-573c-7c91b65a7a7e
-    Description:
-    Local:        false
-    Create Time:  2018-10-22 17:40:24.128188 -0700 PDT
-    Policies:
-        00000000-0000-0000-0000-000000000001 - global-management
-    ```
-
-1.  Configure Vault to connect and authenticate to Consul:
-
-    ```shell-session
-    $ vault write consul/config/access \
-        address="127.0.0.1:8500" \
-        token="7652ba4c-0f6e-8e75-5724-5e083d72cfe4"
-    Success! Data written to: consul/config/access
-    ```
+1.  Configure Vault to connect and authenticate to Consul.
+
+    For Vault 1.11 and above, Vault can bootstrap the Consul ACL system for you if it hasn't already
+    been done. If you have already bootstrapped the Consul ACL system, then you will need to provide
+    Vault with a management token. This can either be the bootstrap token or another management
+    token you've created yourself.    
+
+    1.  Configuring Vault without previously bootstrapping the Consul ACL system:
+
+        ```shell-session
+        $ vault write consul/config/access \
+            address="127.0.0.1:8500"
+        Success! Data written to: consul/config/access
+        ```
+
+        ~> When you let Vault do the bootstrap for you, it will silently store the bootstrap token as the
```

Comment:
Are you intending this to be a note? If so, please apply the format: ~> **Note:**

Reply:
Yup! Added.